src/io/flutter/run/bazel/BazelFields.java

```diff
@@ -27,6 +27,8 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import static io.flutter.run.daemon.RunMode.*;
```

Comment:
Without this import, I get a warning on the switch statement below:

"an enum switch case label must be the unqualified name of an enumeration constant."

Reply:
hmm; sounds like you can't just use `RunMode.PROFILE`? odd. Is that because of a lint we have enabled?

In any case, this isn't a blocker. Feel free to land as is.